src/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(SRC
+    lib/mod_stdlib.f90
```

Comment:
I believe `lib/mod_stdlib.f90` should be deleted in one of the other PR (I believe proposed by @milancurcic ).